src/shared/bus-unit-util.c

```diff
@@ -71,7 +71,7 @@ int bus_parse_unit_info(sd_bus_message *message, UnitInfo *u) {
                         &u->job_path);
 }
 
-static int warn_deprecated(const char *field, const char *eq) {
+static int warn_deprecated(_unused_ sd_bus_message *m, const char *field, const char *eq) {
```

Comment:
why the `_unused_`? we don't need that, do we?

Reply:
It doesn't hurt, and it tells the reader that the param is intentionally unused. We have that in some other places and I think we should do it more.